code/game/objects/items/weapons/nt_melee.dm

```diff
@@ -96,19 +136,112 @@
 		icon_state = initial(icon_state)
 	..()
 
+/obj/item/weapon/tool/sword/nt/scourge/apply_hit_effect(mob/living/carbon/human/target, mob/living/user, hit_zone)
+	. = ..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/O = target
+		target.stun_effect_act(stun, agony, hit_zone, src)
+		O.say(pick("OH", "LORD", "MERCY", "SPARE", "ME", "HAVE", "PLEASE"))
+
+/obj/item/weapon/tool/sword/nt/spear
+	name = "NT Pilum"
+	desc = "A saint looking short spear, designed for use with a shield or as a throwing weapon. The spear-tip usually breaks after being thrown at a target, but it can be welded into shape again."
+	icon_state = "nt_spear"
+	item_state = "nt_spear"
+	wielded_icon = "nt_spear_wielded"
+	force = 24
+	force_wielded_multiplier = 1.08
+	var/tipbroken = FALSE
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = SLOT_BACK | SLOT_BELT 
+	throwforce = 75 
+	armor_penetration = ARMOR_PEN_HALF
+	throw_speed = 3
+	price_tag = 150
+	matter = list(MATERIAL_BIOMATTER = 10, MATERIAL_STEEL = 5) // easy to mass-produce and arm the faithful
+
+/obj/item/weapon/tool/sword/nt/spear/equipped(mob/living/W)
+	. = ..()
+	if(is_held() && is_neotheology_disciple(W))
+		embed_mult = 0.1
+	else
+		embed_mult = initial(embed_mult)
+	if(isBroken)
+		force = WEAPON_FORCE_NORMAL
+		throwforce = WEAPON_FORCE_HARMLESS
+
+/obj/item/weapon/tool/sword/nt/spear/dropped(mob/living/W)
+	embed_mult = 300
```

Comment:
why the crazy embed mult?

Reply:
so it' virtually guaranteed to embed. You're not supposed to be able to reuse it after it hits someone.

Reply:
embeds are super powerfull, it shouldnt be guaranteed.

Reply:
Why do you think they're super powerful? As far as I'm aware, having something embedded inside of you only has a chance to cause pain when you move. Not to mention that, unlike bullet shrapnel, you can yank it out of yourself easily.

Reply:
Constant pain , will easily out-do any painkillers, you can also yank out bullet shrap with a knife , but pulling out embeds deals guaranteed 15 brute + agony and bleeding

Reply:
it doesn't deal constant pain at all, the only effect of an embedded item is having a 5 percent chance per tile to deal 1 to 3 units of brute and toxin damage each if you're in running mode. Having one or even three pilums inside of you will only cause negligible damage and pain.

Reply:
in a battle , its a much more effective form of shrapnel , one that is guaranteed to bring someone down unless they fix it.

Reply:
an embedded sharp item has exactly the same effect as shrapnel while it's in the body. Yes, it deals guaranteed damage when you rip it out, but there isn't much need to do that during the battle as having an item embedded neither deals large amounts of damage nor pain. I encourage you to test this yourself, just yank one or two items into yourself and see for yourself how debiliating it really is.